### ISSUE TITLE
Failed to load resource: net::ERR_FILE_NOT_FOUND when i first "run yarn dev"

### ISSUE THREAD

**Issue by pjshu** [electron-react-boilerplate/electron-react-boilerplate#2341]
## Prerequisites

- [x] Using yarn
- [x] Using node 10.x
- [x] Using an up-to-date [`master` branch](https://github.com/electron-react-boilerplate/electron-react-boilerplate/tree/master)
- [x] Using latest version of devtools. See [wiki for howto update](https://github.com/electron-react-boilerplate/electron-react-boilerplate/wiki/DevTools)
- [ ] Link to stacktrace in a Gist (for bugs)
- [ ] For issue in production release, devtools output of `DEBUG_PROD=true yarn build && yarn start`
- [ ] Tried solutions mentioned in [#400](https://github.com/electron-react-boilerplate/electron-react-boilerplate/issues/400)

## Current Behavior

when i first clone and run "yarn && yarn dev "
it show 

```
Failed to load resource: net::ERR_FILE_NOT_FOUND
renderer.prod.js:1 Failed to load resource: net::ERR_FILE_NOT_FOUND
```
in dev tool  console

and when i  press f5 to refresh
it show 
```
app.html:13 GET file:///home/pjs/test/app/dist/style.css net::ERR_FILE_NOT_FOUND
(anonymous) @ app.html:13
(anonymous) @ app.html:15
app.html:38 GET file:///home/pjs/test/app/dist/renderer.prod.js net::ERR_FILE_NOT_FOUND
```

## Possible Solution

i change package.json
```
"start-main-dev": "cross-env START_HOT=1 NODE_ENV=development electron -r @babel/register ./app/main.dev.js"
```

to

```
"start-main-dev": "cross-env HOT=1 NODE_ENV=development electron -r @babel/register ./app/main.dev.js"
```
all problem solved,
Is this a bug?

- Node version : v10.16.3
- Version or Branch used : master
- Operating System and version : ubuntu 19.10

**Comment by Qquanwei** [comment]
Modify `app>app.html`  `process.env.HOT` -> `process.env.START_HOT` will fixed this issue.

**Comment by pjshu** [comment]
> Modify `app>app.html` `process.env.HOT` -> `process.env.START_HOT` will fixed this issue.

thank you :smile:

**Comment by amilajack** [comment]
Fix was added in `0.18.1`

**Comment by railsjack** [comment]
> > Modify `app>app.html` `process.env.HOT` -> `process.env.START_HOT` will fixed this issue.
> 
> thank you 😄

I love this solution. Thank you very much!
